style.css

```diff
@@ -0,0 +1,976 @@
+}
+
+.hamburger {
+  border: none;
+  background: transparent;
+  width: 30px;
+  height: 20px;
+  position: relative;
+  margin-left: 30px;
+  margin-top: 20px;
+  z-index: 5;
+}
+
+.hamburger:hover {
+  transform: rotate(360deg);
+  transition: all 3s ease-in-out;
+}
+
+.line {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #272a31;
+}
+
+.first-container {
+  background-image: url(images/pexels-laura-penwell-3608056.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  justify-content: space-evenly;
+  flex-direction: column;
+  height: 100%;
+}
+
+.first-image {
+  background-color: rgba(199, 199, 199, 0.82);
+}
+
+.active a {
+  display: block;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  margin-right: 10px;
+  padding-top: 7%;
+  text-decoration: none;
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  font-weight: 700;
+  top: 20px;
+  width: 100%;
+}
+
+.nav-logo {
+  width: 100px;
+  height: 100px;
+  display: none;
+  flex-direction: row;
+  justify-content: center;
+  padding: 20px 100px;
+  position: relative;
+}
+
+.nav-logo img {
+  height: 140px;
+  align-items: center;
+  margin-top: -56px;
+}
+
+span {
+  font-weight: 700;
+  text-decoration: underline;
+}
+
+.container {
+  width: 100%;
+  padding: 30px 10px;
+  align-items: center;
+}
+
+.sec-akara {
+  width: 80%;
+  text-align: center;
+  margin: 20px 40px;
+  padding: 10px 20px;
+}
+
+.etiti-text {
+  width: 80%;
+  text-align: center;
+  margin: 20px 40px;
+  padding: 10px 20px;
+}
+
+.logo-main {
+  border: 2px solid #d3d3d3;
+  display: flex;
+  margin: 40px 10px 20px 10px;
+  align-items: center;
+}
+
+.logo-main img {
+  width: 200px;
+  height: 200px;
+  margin-left: -40px;
+}
+
+.logo-main p {
+  padding-top: 10px;
+  font-size: 30px;
+  font-weight: 700;
+  margin-left: -50px;
+}
+
+.nav-logo a {
+  text-decoration: none;
+  margin-top: 30%;
+  margin-left: -33px;
+  color: #272a31;
+  font-weight: 700;
+  font-family: 'Lato', sans-serif;
+  font-size: 15px;
+  text-align: start;
+  align-items: center;
+  transition: 1s all ease-in-out;
+}
+
+.ul-items li a {
+  text-decoration: none;
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-size: 15px;
+  text-align: center;
+}
+
+.nav-logo a:hover {
+  color: #4d0cf3;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.ul-items li {
+  display: flex;
+  flex-direction: row;
+  margin-right: 50px;
+  margin-top: 3.3%;
+}
+
+.ul-items li a:hover {
+  color: #42ecc4;
+}
+
+.ul-items li:nth-child(5) {
+  border: 3px solid #ec5242;
+  height: 30px;
+  padding: 0 10px;
+  padding-bottom: 10px;
+  margin-top: 15px;
+  text-align: center;
+}
+
+.ul-items {
+  display: block;
+  flex-direction: row;
+  width: 100%;
+  margin-left: 42%;
+}
+
+.first-section {
```

Comment:
- [ ] Kindly make sure the first section on the about section is well centered as in the original template

Your design :point_down: 
![Screenshot from 2023-02-09 22-26-32](https://user-images.githubusercontent.com/97100091/217930408-fb79b582-8b47-4fb2-9035-bd8570efa5f7.png)